content/20.treatments.md

```diff
@@ -498,14 +421,60 @@ The Food and Drug Administration (FDA) has issued warnings to several companies
+Tocilizumab is a humanised monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, de facto inhibiting the IL-6 immune cascade.
+
+**Current Evidence.**
+Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
+Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection [@url:https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741].
+Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April.
+The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyze its efficiency/safety (Biopharma-reporter.com).
+However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial [@doi:10.1093/rheumatology/ker223].
+Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs [@doi:10.1111/1756-185X.13010].
+These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
+This aspect is probably crucial to be investigated further in the trials that are about to start.
+
+Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the COVID-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.
+<!-##### Summary
+
+Summarize the state of the symptom management approach.
+-->
 
 #### Neutralizing Antibodies
 
-Monoclonal antibodies (mAbs) have revolutionized the way we treat human diseases.
+Monoclonal antibodies have revolutionized the way we treat human diseases.
 As a result, they have become some of the best-selling drugs in the pharmaceutical market in recent years [@doi:10.1186/s12929-019-0592-z].
 There are currently 79 FDA approved mAbs on the market including antibodies for viral infections (e.g. Ibalizumab for HIV and Palivizumab for RSV) [@doi:10.1186/s12929-019-0592-z; @doi:10.1146/annurev-immunol-032712-095916].
-Although vaccines remain the most important way to prevent viral infections, their development process is long and they fail to provide immediate prophylactic protection or treat ongoing infections [@doi:10.3390/v10120680].
```

Comment:
Why do you suggest to delete that?

Reply:
I moved it to the first, more introductory part:

"While vaccines would be expected to save the largest number of lives by bolstering the immune response of the at-risk population broadly to the virus, resulting in a lower rate of infection, their development process is long and they fail to provide immediate prophylactic protection or treat ongoing infections [@doi:10.3390/v10120680]."

The pluses and minuses of vaccines were mentioned throughout the section, so I tried to consolidate things a bit more. To me at least it made sense to have that part when introducing the section and not repeat it in multiple places.